### ISSUE TITLE
Add GH action for dispatching release event

### ISSUE THREAD

**Issue by sigalsax** [cyberark/secrets-provider-for-k8s#239]
As part of our Secrets Provider release flow, our helm charts are packaged   as part of our pipeline build and are saved to the artifacts tab in Jenkins. We then need to manually take the artifact and create a PR in a [separate repo](https://github.com/cyberark/helm-charts). 

The act of pushing the artifact to a separate repo can be automated. In essence we want to be able to dispatch eventA from repoA (SP) in order to trigger an actionA in another repoB (helm-charts).

EventA - a release
ActionA - fetch artifacts from Jenkins build of tagged release build

More detailed explanation provided [here](https://github.com/cyberark/community/issues/77)
```
GIVEN I am releasing suite component A
AND the release of suite component A produces an artifact A1
AND the artifact A1 must be uploaded to suite component B on release of suite component A
THEN there are Jenkins pipeline and/or GitHub action utilities that can copy the component A pipeline-built artifact A1 to a folder on a branch of component B
AND there is a Jenkins pipeline and/or GitHub action utility to open a PR to merge the branch inc
```

To achieve this see the following resources:
1️⃣  https://github.community/t/triggering-by-other-repository/16163
2️⃣ https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#repository_dispatch
3️⃣  https://github.community/t/github-workflow-trigger-from-another-repository/17461
4️⃣  https://github.community/t/trigger-a-workflow-from-another-workflow/17395

TL;DR we need two actions one for dispatching an event and the other for setting a trigger.

Tutorial 101 example of sending and receiving events [here](https://github.com/marketplace/actions/repository-dispatch) and [this](https://dev.to/teamhive/triggering-github-actions-using-repository-dispatches-39d1) (high recommended)

**Comment by izgeri** [comment]
Noting here that this may be relevant to cyberark/community#77
